src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1071,9 +1073,10 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
             std::weak_ptr<PeerImp> weak = shared_from_this();
             app_.getJobQueue ().addJob (
                 jtTRANSACTION, "recvTransaction->checkTransaction",
-                [weak, flags, stx] (Job&) {
+                [weak, flags, checkSignature, stx] (Job&) {
```

Comment:
We currently *allow* `RIPPLED_USE_CPP_14=1`.  With this change we would *require* `RIPPLED_USE_CPP_14=1`.  I'm ok with that.  I'm testing develop with that now just to double check it.  I'm just saying this to ensure that we don't make this happy and momentous move accidentally, but with eyes wide open.